src/SFA.DAS.EmployerFinance.Web/Controllers/HomeController.cs

```diff
@@ -37,6 +38,14 @@ public ActionResult Privacy()
             return Redirect(Url.LegacyEasAction("service/privacy"));
         }
 
+        [HttpGet]
+        [Route("{HashedAccountId}/cookieConsent", Order = 0)]
+        [Route("cookieConsent", Order = 1)]
+        public ActionResult CookieConsent()
```

Comment:
Did we need to create a controller action? Other places we just point the link directly to the relevant url in EmployerAccounts

Reply:
Its to keep the consistency with the they way it works for privacy.